src/Microsoft.Extensions.Logging.Abstractions/LogFormatter.cs

```diff
@@ -27,7 +27,7 @@ public static string Formatter(object state, Exception e)
             var result = string.Empty;
             if (state != null)
             {
-                var values = state as ILogValues;
+                var values = state as IReadOnlyList<KeyValuePair<string, object>>;
```

Comment:
Hmmm... I didn't notice this before. It turns out the entire LogFormatter class should be deleted - there is no case where we should have a standard name-value string builder which appends the exception with a newline. Can you delete this file and see how much depended on it?